### PR TITLE
Task 1: Frameworks

### PR DESCRIPTION
**Pull request by Ellyia**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "main": "src/index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon index.js",
+    "start:dev": "nodemon --watch src"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "express": "^4.18.2",
```
Comment (vasylkivskyy) [inline package.json:14]: nodemon should be in the devDependencies section as it is using only for local development, others need to be as dependencies

File: src/routes/userRoute.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+const express = require('express');
```
Comment (vasylkivskyy) [inline src/routes/userRoute.js:1]: As I understand, these routes are not connected in the main file - index.js

File: swagger.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    title: 'API Documentation',
+    version: '1.0.0',
+    description: 'API documentation for Ella`s Node.js application',
+  },
+  basePath: '/',
+};
+
+const options = {
+  swaggerDefinition,
+  apis: ['./routes/*.js'],
```
Comment (vasylkivskyy) [inline swagger.js]: Is this the correct file path? Swagger.js file is outside the src folder.

File: package.json (hunk 1 above)
Comment (Ellyia) [reply package.json:14]: changed

File: src/routes/userRoute.js (hunk 2 above)
Comment (Ellyia) [reply src/routes/userRoute.js:1]: changed and connected

File: swagger.js (hunk 3 above)
Comment (Ellyia) [reply swagger.js]: I have changed the file path, thank you

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+const { swaggerUi, swaggerSpec } = require('../swagger');
+const errorHandler = require('./middlewares/errorHandlingMiddleware');
+const routes = require('./routes/routes');
+
+const app = express();
+const PORT = 3000; 
+ 
+app.use('/api', routes); 
+app.use('/api-docs', swaggerUi.serve, swaggerUi.setup(swaggerSpec));
+
```
Comment (vasylkivskyy) [inline src/index.js:11]: It is also good practice to add notFoundMiddleware

**Review by vasylkivskyy — APPROVED** [review]

File: src/index.js (hunk 4 above)
Comment (Ellyia) [reply src/index.js:11]: Thank you, I will add this changes in the next PR

**Pull request merged**